docbrown/src/db/graph_window.rs

```diff
@@ -798,6 +801,71 @@ impl<G: GraphViewOps> GraphViewInternalOps for WindowedGraph<G> {
         )
     }
 
+    fn temporal_vertex_timestamps_vec(
```

Comment:
Just on the naming of these - I don't think we need them to start with temporal as we don't have a static equivalent and we can probably remove vec at the end as I think they will always be a vector. I.e.   `temporal_vertex_timestamps_vec` -> `vertex_timestamps` ---`temporal_vertex_timestamps_vec_window` -> `vertex_timestamps_window`

Reply:
Across all the levels I mean, should make these a bit less of a mouth full